scripts/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+TAG=$( cd src/github.com/SUSE/cf-plugin-backup
+       git describe --tags --long | awk -F - '{ print $1 }' )
```

Comment:
This can be written (assuming we have a new enough git) as
```bash
git -C src/github.com/SUSE/cf-plugin-backup describe --tags --long | awk -F - '{ print $1 }'
```

Reply:
Will try